--- /mingw64/include/float.h.old	2023-09-13 20:03:21.000000000 +0200
+++ /mingw64/include/float.h	2023-09-25 15:52:26.235629300 +0200
@@ -149,6 +149,7 @@
 #define	_MCW_IC		0x00040000	/* Infinity */
 #define	_MCW_RC		0x00000300	/* Rounding */
 #define	_MCW_PC		0x00030000	/* Precision */
+#define	MCW_PC		0x00030000	/* Precision */
 
 /* Number of base-FLT_RADIX digits in the significand, p.  */
 #undef FLT_MANT_DIG
